config/rbac/auth_proxy_role.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: proxy-role
```

Since we are using "proxy" in multiple contexts, we should be more specific:

```suggestion
  name: metrics-proxy-role
```